components/NewOrder/UploadImageForm.js

```diff
@@ -129,40 +100,18 @@ const UploadImageForm = ({ FadeIn, images, setImages }) => {
             justifyContent: "space-around",
           }}
         >
-          <Button
-            textColor="#14213D"
-            icon="camera"
-            onPress={() => captureImage()}
-          >
+          <Button textColor="#14213D" icon="camera" onPress={() => handleCaptureImage()}>
```

Comment:
onPress={() => handleCaptureImage()} can be shortened to just onPress={handleCaptureImage}